pkg/paho-mqtt/patches/0001-mods-to-RIOT-building.patch

```diff
@@ -0,0 +1,20 @@
+From 54b50b97e3b23159808c44311a7ba474909f3e2e Mon Sep 17 00:00:00 2001
+From: Javier FILEIV <javier.fileiv@gmail.com>
+Date: Mon, 21 Oct 2019 13:50:59 +0200
+Subject: [PATCH] mods to RIOT building
+
+
+diff --git a/MQTTClient-C/src/MQTTClient.c b/MQTTClient-C/src/MQTTClient.c
+index bd24dff..4326021 100755
+--- a/MQTTClient-C/src/MQTTClient.c
++++ b/MQTTClient-C/src/MQTTClient.c
+@@ -363,7 +363,7 @@ int MQTTIsConnected(MQTTClient* client)
+   return client->isconnected;
+ }
+ 
+-void MQTTRun(void* parm)
++void *MQTTRun(void* parm)
```

Comment:
This can be done without a patch, by making the thread-handler part of `riot_iface.c` and just calling `MQTTRun()` from it

```C
void *_mqtt_run(void *args)
{
    MQTTRun(args);  /* never returns */
    return NULL
}
```

Patches should be avoided for packages, if possible.